hypervisor/debug/shell.c

```diff
@@ -518,6 +515,11 @@ static int32_t shell_cmd_help(__unused int32_t argc, __unused char **argv)
 		int32_t i = 0;
 		uint32_t j;
 
+		snprintf(str, MAX_STR_SIZE, "  %-15s %-53s %-s",
```

Comment:
The output won't look very good on an 80-character terminal (you've consumed 72 characters before you get to the description).  I'm not sure what the supported terminal width requirement should be, but the usual is 80 characters and your output has lines that are more than 200 characters wide.  Line wrapping will make this output confusing to read (DX issue).

Reply:
@dbkinder  Ok, just now i have a test with picocom, there have line wrapping when the output is bigger than 80, i will have a change with this. thanks